changelogs/README.md

```diff
@@ -0,0 +1,5 @@
+# Revised Changelog and Release Notes Process
+
+For information regarding the revised changelog and release notes process, please consult the README in the GitHub Actions repository that this process utilizes. To view this README, follow the link below:
```

Comment:
```suggestion
For information regarding the changelog and release notes process, please consult the README in the GitHub Actions repository that this process utilizes. To view this README, follow the link below:
```

No need to mention revised. This will be the process going forward, so we can just call it that.

Reply:
Updated!